[2025-06-24T00:23:33.091+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-24T00:23:33.110+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-23T17:23:30.724556+00:00 [queued]>
[2025-06-24T00:23:33.115+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-23T17:23:30.724556+00:00 [queued]>
[2025-06-24T00:23:33.115+0700] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-06-24T00:23:33.130+0700] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): start_zookeeper> on 2025-06-23 17:23:30.724556+00:00
[2025-06-24T00:23:33.136+0700] {standard_task_runner.py:63} INFO - Started process 18327 to run task
[2025-06-24T00:23:33.140+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'start_kafka_hadoop_and_run_notebooks', 'start_zookeeper', 'manual__2025-06-23T17:23:30.724556+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/start_DAG.py', '--cfg-path', '/tmp/tmplz9z8ttx']
[2025-06-24T00:23:33.141+0700] {standard_task_runner.py:91} INFO - Job 36: Subtask start_zookeeper
[2025-06-24T00:23:33.182+0700] {task_command.py:426} INFO - Running <TaskInstance: start_kafka_hadoop_and_run_notebooks.start_zookeeper manual__2025-06-23T17:23:30.724556+00:00 [running]> on host Panda
[2025-06-24T00:23:33.265+0700] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group_03' AIRFLOW_CTX_DAG_ID='start_kafka_hadoop_and_run_notebooks' AIRFLOW_CTX_TASK_ID='start_zookeeper' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T17:23:30.724556+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-23T17:23:30.724556+00:00'
[2025-06-24T00:23:33.266+0700] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-24T00:23:33.267+0700] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-24T00:23:33.267+0700] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        cd /home/panda/kafka_2.13-3.9.1 &&         nohup bin/zookeeper-server-start.sh config/zookeeper.properties > /tmp/zookeeper.log 2>&1 &\n        sleep 5 && exit 0\n        ']
[2025-06-24T00:23:33.282+0700] {subprocess.py:86} INFO - Output:
[2025-06-24T00:33:18.583+0700] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-06-24T00:33:18.583+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-06-24T00:33:18.596+0700] {process_utils.py:132} INFO - Sending Signals.SIGTERM to group 18327. PIDs of all processes in the group: [18328, 18327]
[2025-06-24T00:33:18.596+0700] {process_utils.py:87} INFO - Sending the signal Signals.SIGTERM to group 18327
[2025-06-24T00:33:18.596+0700] {taskinstance.py:2607} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-06-24T00:33:18.597+0700] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2025-06-24T00:33:18.598+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-24T00:33:18.649+0700] {process_utils.py:80} INFO - Process psutil.Process(pid=18328, status='terminated', started='00:23:32') (18328) terminated with exit code None
[2025-06-24T00:33:18.649+0700] {process_utils.py:80} INFO - Process psutil.Process(pid=18327, status='terminated', exitcode=0, started='00:23:32') (18327) terminated with exit code 0
